store/store_test.go

```diff
@@ -0,0 +1,263 @@
+package store
+
+import "testing"
+import "time"
+
+//import "strconv"
+//import "fmt"
+//import "golang.org/x/crypto/blake2b"
+//import "encoding/hex"
+
+func TestItemsAdd(t *testing.T) {
+	db, _ := NewDatabase(time.Second*86400, time.Second*3600, time.Second*86400)
+
+	testVal := "q"
+
+	var testNodeID NodeID
+	copy(testNodeID[:], "w")
+
+	err := db.AddItem(testVal, testNodeID)
+	if err != nil {
+		t.Errorf("some error in AddItem that is not yet defined")
```

Comment:
So `db.AddItem` cannot error? Just remove the return type instead and skip this check here